### PR TITLE
Add nginx and docker to supervisor with instructions to autostart

### PR DESCRIPTION
**Pull request by yrkumar**
This addresses some parts of #121. More specifically, it adds nginx and docker to supervisor. Exposing the ports of the Tango service in the docker container simply involves adding extra params when running the docker container so that has been added to the [suggested deployment documentation](https://github.com/autolab/Tango/wiki/Suggested-Deployment).

### CODE REVIEW

**Comment by yrkumar** [comment]
@nayak16 does this look good to merge?

File: Dockerfile [hunk 1]
```diff
@@ -53,22 +52,21 @@ RUN chmod +x /usr/local/bin/wrapdocker
 
 # Define additional metadata for our image.
 VOLUME /var/lib/docker
-CMD ["wrapdocker"]
```
Comment (nayak16) [inline Dockerfile]: Why was this removed? @yrkumar

Comment (yrkumar) [inline Dockerfile]: `docker ps` seems to work inside the docker container even without running this wrapper script so I feel more comfortable leaving it out. It seems to be a wrapper to start docker, which won't work in the `Dockerfile` anyways since you can only run one command at the end of the `Dockerfile` to be run in the container. 

As an alternative, I can set `wrapdocker` as the process to start in `supervisor` instead of Docker itself, which will ensure that `wrapdocker` runs (and thus will start Docker). Is this preferred?

**Comment by nayak16** [comment]
Let's merge this. It looks good

**Comment by yrkumar** [comment]
Closed by b452d05848e09628720e57a8232d53a1d2eadb25
